### DISCUSSION TITLE
https://theme-hope.vuejs.press/zh/guide/layout/home.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#591] [Announcements]
# https://theme-hope.vuejs.press/zh/guide/layout/home.html

vuepress-theme-hope 保留了默认主题的首页配置，在此基础上改进样式与配置。 如果想要使用它，请在 frontmatter 中设置 home: true。任何 frontmatter 之后的内容将会以普通的 Markdown 被渲染，并插入到主页渲染内容的底部。

https://theme-hope.vuejs.press/zh/guide/layout/home.html

**Comment by JJlinCN** [comment]
首页之后的Markdown，新版本首个一级标题会被吞，有占据空间但是不显示，后续标题恢复正常，不清楚是写法不对还是有Bug，反馈一下

**Reply by Mister-Hope** [reply]
根据 a11y 规范，一个网页应该只能有一个一级标题，已经在主页上方显示过了。

所以我不认为这是bug。

**Reply by Mister-Hope** [reply]
换言之，你本就不应该再主页之后的内容使用 h1

**Comment by JJlinCN** [comment]
这是来自QQ邮箱的假期自动回复邮件。彭粤庭已收到

**Comment by 006hjy** [comment]
鼠标移入和鼠标移出title时的炫光效果是怎么做的？我用一样的代码，没有炫光效果。

**Reply by Mister-Hope** [reply]
一个预设

**Comment by QingFeng-awa** [comment]
heroText不能自定义颜色,只能根据主题色生成吗

**Reply by Mister-Hope** [reply]
有没有可能文档教你自定义样式，而且你可以很轻松的写一个css把它覆盖掉？

**Comment by Triabin** [comment]
大佬，features有打算支持固定卡片大小吗？
